src/core/components/model-example.jsx

```diff
@@ -58,42 +60,83 @@ export default class ModelExample extends React.Component {
     let { defaultModelExpandDepth } = getConfigs()
     const ModelWrapper = getComponent("ModelWrapper")
     const HighlightCode = getComponent("highlightCode")
+    const exampleTabId = randomBytes(5).toString("base64")
+    const examplePanelId = randomBytes(5).toString("base64")
+    const modelTabId = randomBytes(5).toString("base64")
+    const modelPanelId = randomBytes(5).toString("base64")
 
     let isOAS3 = specSelectors.isOAS3()
 
-    return <div className="model-example">
-      <ul className="tab">
-        <li className={ "tabitem" + ( this.state.activeTab === "example" ? " active" : "") }>
-          <a className="tablinks" data-name="example" onClick={ this.activeTab }>{isExecute ? "Edit Value" : "Example Value"}</a>
-        </li>
-        { schema ? <li className={ "tabitem" + ( this.state.activeTab === "model" ? " active" : "") }>
-          <a className={ "tablinks" + ( isExecute ? " inactive" : "" )} data-name="model" onClick={ this.activeTab }>
-            {isOAS3 ? "Schema" : "Model" }
-          </a>
-        </li> : null }
-      </ul>
-      <div>
-        {
-          this.state.activeTab === "example" ? (
-            example ? example : (
+    return (
+      <div className="model-example">
+        <ul className="tab" role="tablist">
+          <li className={cx("tabitem", { "active": this.state.activeTab === "example" })} role="presentation">
+            <button
+              aria-controls={examplePanelId}
+              aria-selected={this.state.activeTab === "example"}
+              className="tablinks"
+              data-name="example"
+              id={exampleTabId}
+              onClick={ this.activeTab }
+              role="tab"
+            >
+              {isExecute ? "Edit Value" : "Example Value"}
+            </button>
+          </li>
+          { schema && (
+            <li className={cx("tabitem", { "active": ["model", "schema"].includes(this.state.activeTab) })} role="presentation">
+              <button
+                aria-controls={modelPanelId}
+                aria-selected={["model", "schema"].includes(this.state.activeTab)}
+                className={ "tablinks" + ( isExecute ? " inactive" : "" )}
+                data-name={isOAS3 ? "schema" : "model"}
```

Comment:
Do we actually need to distinguish here? If we would call this `schema-or-model` wouldn't this simplify the code?

Reply:
Yes! I wasn't sure if this was intentional so I wanted to keep everything as compatible as possible. If keeping it as one or the other would suffice just let me know what the preference is

Reply:
The original code has always used `model` class and `model`  in `data-name` attribute right? Can we get the same thing here?

Reply:
All set!